benchmarks/startup/config.yaml

```
@@ -0,0 +1,25 @@
+ids_only: &defaults
```

this will get run as well, so you might want to do:

```suggestion
startup_with_ddtrace_run: &defaults
```